### PR TITLE
Release documentation update

### PR DESCRIPTION
**Pull request by bryantaustin13**
This answers issue #216 first point:
Would be good to explain the significance of the minor and patch versions:

major: Guideline version
minor: New features and capabilities, but not breaking changes
patch: Bug fixes and technical corrections.
Would be good to illustrate that in the diagram with a 2022-Fix branch and 2016-Fix branch showing the version increment

Snd the point:
For 2022, the IG and all the artifacts in it are "draft" status. 
For 2016,, the IG is active, and any artifact we have pilot experience with is "active", all the rest are "draft"

Other parts of the issue forthcoming.

### CODE REVIEW

**Review by c-schuler — APPROVED** [review]
Looks good to me. I updated the homepage and footer with items discussed in the touchpoint.

File: README.md [hunk 1]
```diff
@@ -34,15 +34,21 @@ The HL7 IG Publisher is committed to this repository to make building as easy as
 
 ## Release and Branching
 
-This repository will leverage a branching strategy to maintain multiple versions of opioid recommendations (currently 2016 and 2022) and their respective IG's. This will use semantic versioning on releases to make it clear which version of the recommendations the release is for. Any release for 2022 version will be v2022.#.# and any that are 2016 will be v2016.#.#. 
+This repository will leverage a branching strategy to maintain multiple versions of the CDC opioid recommendations (currently 2016 and 2022) and their respective IG's. This will use semantic versioning on releases to make it clear which version of the recommendations the release is for.
+ The format used will be major.minor.patch
+- major is for new CDC Recommendations
```
Comment (brynrhodes) [inline README.md:39]: Consider: major is used to refer to the guideline publication year (e.g. 2016)

File: README.md [hunk 2]
```diff
@@ -34,15 +34,21 @@ The HL7 IG Publisher is committed to this repository to make building as easy as
...
+- minor is for new features and capabilities, but not breaking changes, thus backwards compatible
+- patches are bug fixes and technical corrections
 
-This will be developed from source branches for each version, master will be 2022 latest development, and v2016 will be latest development for 2016. This will leverage a gitflow type model where feature branches are made from respective branches and merged back to the parent via pull request and approved peer review. See diagram below
+Any release for 2022 version will be v2022.#.# and any that are 2016 will be v2016.#.#.
```
Comment (brynrhodes) [inline README.md:44]: Consider adding: Note that the `v` prefix is only used in the branch and tag names in git. The actual version will be `2016.#.#`

**Review by brynrhodes — CHANGES_REQUESTED** [review]
Looks great, just a few minor wording tweaks

File: README.md [hunk 3]
```diff
@@ -34,15 +34,22 @@ The HL7 IG Publisher is committed to this repository to make building as easy as
 
 ## Release and Branching
 
-This repository will leverage a branching strategy to maintain multiple versions of opioid recommendations (currently 2016 and 2022) and their respective IG's. This will use semantic versioning on releases to make it clear which version of the recommendations the release is for. Any release for 2022 version will be v2022.#.# and any that are 2016 will be v2016.#.#. 
+This repository will leverage a branching strategy to maintain multiple versions of the CDC opioid recommendations (currently 2016 and 2022) and their respective IG's. This will use semantic versioning on releases to make it clear which version of the recommendations the release is for.
+ The format used will be major.minor.patch
+- major is for new CDC Recommendations and refers to the guideline publication year (e.g. 2016)
+- minor is for new features and capabilities, but not breaking changes, thus backwards compatible
```
Comment (brynrhodes) [inline README.md:40]: major represents the guideline version, and refers to the publication year (e.g. 2016)

Comment (brynrhodes) [reply README.md:40]: Specifically, the wording here is saying "CDC Recommendations", it's not that, it's the "guideline" version.

**Review by brynrhodes — CHANGES_REQUESTED** [review]

**Review by brynrhodes — APPROVED** [review]

**Pull request merged**